zluda/src/impl/graph.rs

```diff
@@ -8,6 +9,48 @@ pub(crate) unsafe fn exec_destroy(graph_exec: hipGraphExec_t) -> hipError_t {
     hipGraphExecDestroy(graph_exec)
 }
 
+pub(crate) fn exec_update_v2(
+    h_graph_exec: hipGraphExec_t,
+    h_graph: hipGraph_t,
+    result_info: &mut cuda_types::cuda::CUgraphExecUpdateResultInfo,
+) -> hipError_t {
+    let mut h_error_node: hipGraphNode_t = unsafe { std::mem::zeroed() };
+    let mut update_result: hipGraphExecUpdateResult = unsafe { std::mem::zeroed() };
+    unsafe { hipGraphExecUpdate(h_graph_exec, h_graph, &mut h_error_node, &mut update_result) }?;
+
+    result_info.errorNode = unsafe { std::mem::transmute(h_error_node) };
```

Comment:
Is there a way to use `FromCuda` instead of transmute here and below?
In case we change the definition of CUgraphExec to be more complicated than transpose `hipGraphExec_t`